### PR TITLE
Added online Server to build

### PR DESCRIPTION
**Pull request by MathiasSchjoedt-Bavngaard**

### CODE REVIEW

File: Web/CCMS/package.json
Comment (MathiasSchjoedt-Bavngaard) [inline Web/CCMS/package.json:1]: added to commands to make actions run correct scripts

File: .github/workflows/CreditCardDeployment.yml [hunk 1]
```diff
@@ -75,10 +75,6 @@ jobs:
       - name: Install dependencies
         run: npm install
         working-directory: ./Web/CCMS
-
-      - name: Install Angular gh-pages // skip-confirmation
```
Comment (MathiasSchjoedt-Bavngaard) [inline .github/workflows/CreditCardDeployment.yml:79]: is already installed in dependensies

File: .github/workflows/CreditCardDeployment.yml [hunk 2]
```diff
@@ -49,7 +49,7 @@ jobs:
         working-directory: ./Web/CCMS
 
       - name: build to prod
-        run: npm run ng build --base-href=https://Sirety.github.io/AFE/
+        run: npm run build:prod
```
Comment (MathiasSchjoedt-Bavngaard) [inline .github/workflows/CreditCardDeployment.yml:52]: now in package jason

File: credit-card-server/src/index.ts
Comment (MathiasSchjoedt-Bavngaard) [inline credit-card-server/src/index.ts:1]: just formatting

File: credit-card-server/package.json [hunk 3]
```diff
@@ -5,15 +5,19 @@
   "main": "index.ts",
   "scripts": {
     "build": "tsc",
-    "start": "tsc && node dist/index.js",
+    "start": "node dist/index.js",
```
Comment (MathiasSchjoedt-Bavngaard) [inline credit-card-server/package.json:8]: skipping build when starting

**Review by MathiasSchjoedt-Bavngaard — COMMENTED** [review]
Should be working on https://sirety.github.io/AFE/creditcard

**Review by Sirety — APPROVED** [review]

**Pull request merged**